### PR TITLE
multiple pages: replace all die.net links

### LINKED ISSUE #5143: Replace die.net links with better resources where applicable
There was [a short discussion](https://gitter.im/tldr-pages/tldr?at=600008e514cec811ec998765) today on Gitter about pages which have "more information" links pointing to `die.net`. As @navarroaxel showed there is man.archlinux.org, which is for Arch Linux programs, and which could be better suited for Arch Linux specific utilities.

In general though, we have the same distro-generic set of manual pages that `die.net` offers on `man7.org`, which IMHO renders the pages a lot better (sometimes `die.net` pages have syntax errors due to special characters being treated strangely when rendered in HTML, furthermore `man7.org` pages are actually in monospace). 

Therefore I suggest we convert all "more information" links which are pointing to `die.net` to point to `man7.org` instead, and in case the package/command is distro-specific, directly point to the distribution's own manual pages if available, for example:

- [Ubuntu](http://manpages.ubuntu.com/manpages/)
- [Debian](https://manpages.debian.org/buster/man-db/index.html)
- [Arch](https://man.archlinux.org)
- etc...

### PR DESCRIPTION
**Pull request by marchersimon**
Closes #5143

### CODE REVIEW

**Comment by marchersimon** [comment]
There's an arch man page for `pdflatex`, but it covers only `pdftex`. However, those commands do have the exact same arguments, and if you execute `man pdflatex`, you also get to the manpage of `pdftex`

File: pages.fr/linux/ifdown.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ifdown
 
 > Désactive des interfaces réseau.
-> Plus d'information : <https://linux.die.net/man/8/ifdown>.
+> Plus d'information : <https://manpages.debian.org/jessie/ifupdown/ifdown.8.en.html>.
```
Comment (navarroaxel) [inline pages.fr/linux/ifdown.md:4]: We can remove the i18n here:
```suggestion
> Plus d'information : <https://manpages.debian.org/jessie/ifupdown/ifdown.8.html>.
```

File: pages.fr/linux/ifup.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # ifup
 
 > Outil utilisé pour activer des interfaces réseau.
-> Plus d'information : <https://linux.die.net/man/8/ifup>.
+> Plus d'information : <https://manpages.debian.org/jessie/ifupdown/ifup.8.en.html>.
```
Comment (navarroaxel) [inline pages.fr/linux/ifup.md:4]:
```suggestion
> Plus d'information : <https://manpages.debian.org/latest/ifupdown/ifup.8.html>.
```

File: pages.zh/linux/logsave.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # logsave
 
 > 将一个命令的输出保存在日志文件中.
-> 更多信息: <https://linux.die.net/man/8/logsave>.
+> 更多信息: <https://man.archlinux.org/man/core/e2fsprogs/logsave.8.en>.
```
Comment (navarroaxel) [inline pages.zh/linux/logsave.md:4]: We can remove the i18n here:
```suggestion
> 更多信息: <https://man.archlinux.org/man/core/e2fsprogs/logsave.8>.
```

File: pages/linux/ifdown.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # ifdown
 
 > Disable network interfaces.
-> More information: <https://linux.die.net/man/8/ifdown>.
+> More information: <https://manpages.debian.org/jessie/ifupdown/ifdown.8.en.html>.
```
Comment (navarroaxel) [inline pages/linux/ifdown.md:4]: We can remove the i18n and the debian version here:
```suggestion
> More information: <https://manpages.debian.org/latest/ifupdown/ifdown.8.html>.
```

File: pages/linux/ifup.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # ifup
 
 > Tool used to enable network interfaces.
-> More information: <https://linux.die.net/man/8/ifup>.
+> More information: <https://manpages.debian.org/jessie/ifupdown/ifup.8.en.html>.
```
Comment (navarroaxel) [inline pages/linux/ifup.md:4]:
```suggestion
> More information: <https://manpages.debian.org/latest/ifupdown/ifup.8.html>.
```

File: pages/linux/iwconfig.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # iwconfig
 
 > Configure and show the parameters of a wireless network interface.
-> More information: <https://linux.die.net/man/8/iwconfig>.
+> More information: <https://man.archlinux.org/man/core/wireless_tools/iwconfig.8.en>.
```
Comment (navarroaxel) [inline pages/linux/iwconfig.md:4]:
```suggestion
> More information: <https://man.archlinux.org/man/core/wireless_tools/iwconfig.8>.
```

File: pages/linux/logsave.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # logsave
 
 > Save the output of a command in a logfile.
-> More information: <https://linux.die.net/man/8/logsave>.
+> More information: <https://man.archlinux.org/man/core/e2fsprogs/logsave.8.en>.
```
Comment (navarroaxel) [inline pages/linux/logsave.md:4]:
```suggestion
> More information: <https://man.archlinux.org/man/core/e2fsprogs/logsave.8>.
```

File: pages/linux/snmpwalk.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 # snmpwalk
 
 > SNMP query tool.
-> More information: <https://linux.die.net/man/1/snmpwalk>.
+> More information: <https://man.archlinux.org/man/extra/net-snmp/snmpwalk.1.en>.
```
Comment (navarroaxel) [inline pages/linux/snmpwalk.md:4]:
```suggestion
> More information: <https://man.archlinux.org/man/extra/net-snmp/snmpwalk.1>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.fr/linux/ifdown.md [hunk 9]
```diff
@@ -1,7 +1,7 @@
 # ifdown
 
 > Désactive des interfaces réseau.
-> Plus d'information : <https://manpages.debian.org/jessie/ifupdown/ifdown.8.en.html>.
+> Plus d'information : <https://manpages.debian.org/jessie/ifupdown/ifdown.8.html>.
```
Comment (navarroaxel) [inline pages.fr/linux/ifdown.md:4]:
```suggestion
> Plus d'information : <https://manpages.debian.org/latest/ifupdown/ifdown.8.html>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by marchersimon** [comment]
Actually, it's done. I'm just not sure if it's fine that I used `man.archlinux.org` pages where possible.

**Review by bl-ue — APPROVED** [review]

**Comment by waldyrious** [comment]
> I'm just not sure if it's fine that I used `man.archlinux.org` pages where possible.

@bl-ue didn't we decide recently that we'd favor man7.org links for manpages, instead of distro-specific ones? Or is there a reason to go with ArchLinux manpages instead?

**Comment by marchersimon** [comment]
I decided for the arch man pages mostly because of https://github.com/tldr-pages/tldr/issues/5143#issuecomment-808998349.

**Comment by marchersimon** [comment]
Also, the ArchLinux manpages often are available in multiple languages

**Comment by waldyrious** [comment]
@marchersimon D'oh! 🤦 — the link to the relevant issue was right there in the opening comment of the PR, sorry for my clumsiness. Yeah, that's what I had in mind. However, it feels like the discussion there isn't closed yet. Let's hold this PR until we reach a consensus in that thread. Please comment there if you have additional points to add (such as the multilingual support), and in the meantime I'll ping some people to respond there.

**Comment by marchersimon** [comment]
Sure, thanks

File: pages/common/pdflatex.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # pdflatex
 
 > Compile a PDF document from LaTeX source files.
-> More information: <https://linux.die.net/man/1/pdflatex>.
+> More information: <https://www.unix.com/man-page/redhat/1/pdflatex>.
```
Comment (mebeim) [inline pages/common/pdflatex.md:4]: I saw your commend above about this, and I think we can point this to Arch man pages even if the Arch link says `pdftex`. After all, even this page documents `pdftex`, so I don't see a problem.

File: pages.de/common/pdflatex.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 # pdflatex
 
 > Kompiliere eine LaTeX Quelldatei in ein PDF Dokument.
-> Mehr Informationen: <https://linux.die.net/man/1/pdflatex>.
+> Mehr Informationen: <https://www.unix.com/man-page/redhat/1/pdflatex>.
```
Comment (mebeim) [inline pages.de/common/pdflatex.md:4]: Same here.

File: pages/linux/e2image.md [hunk 12]
```diff
@@ -1,7 +1,7 @@
 # e2image
 
 > Save critical ext2/ext3/ext4 filesystem metadata to a file.
-> More information: <https://linux.die.net/man/8/e2image>.
+> More information: <https://man7.org/linux/man-pages/man8/e2image.8.html>.
```
Comment (mebeim) [inline pages/linux/e2image.md:4]: Looks like there is `https://man.archlinux.org/man/e2image.8.en`, we can use this one here.

```suggestion
> More information: <https://man.archlinux.org/man/e2image.8.en>.
```

File: pages/linux/pmount.md [hunk 13]
```diff
@@ -1,7 +1,7 @@
 # pmount
 
 > Mount arbitrary hotpluggable devices as a normal user.
-> More information: <https://linux.die.net/man/1/pmount>.
+> More information: <https://www.unix.com/man-page/suse/1/pmount>.
```
Comment (mebeim) [inline pages/linux/pmount.md:4]: It seems like `pmount` has an Ubuntu manual page:

```suggestion
> More information: <http://manpages.ubuntu.com/manpages/focal/en/man1/pmount.1.html>.
```

File: pages/linux/semanage.md [hunk 14]
```diff
@@ -1,7 +1,7 @@
 # semanage
 
 > SELinux Policy Management tool.
-> More information: <https://linux.die.net/man/8/semanage>.
+> More information: <https://man7.org/linux/man-pages/man8/semanage.8.html>.
```
Comment (mebeim) [inline pages/linux/semanage.md:4]: Same here:

```suggestion
> More information: <http://manpages.ubuntu.com/manpages/focal/en/man8/semanage.8.html>.
```

**Review by mebeim — CHANGES_REQUESTED** [review]
Thanks for the work. Also, good job on spotting that `lftp` manual page. I've found some better alternatives, see below.

File: pages/linux/semanage.md [hunk 15]
```diff
@@ -1,7 +1,7 @@
 # semanage
 
 > SELinux Policy Management tool.
-> More information: <https://linux.die.net/man/8/semanage>.
+> More information: <http://manpages.ubuntu.com/manpages/latest/man8/semanage.8.html>.
```
Comment (navarroaxel) [inline pages/linux/semanage.md:4]:
```suggestion
> More information: <https://manpages.ubuntu.com/manpages/latest/man8/semanage.8.html>.
```

File: pages/linux/pmount.md [hunk 16]
```diff
@@ -1,7 +1,7 @@
 # pmount
 
 > Mount arbitrary hotpluggable devices as a normal user.
-> More information: <https://linux.die.net/man/1/pmount>.
+> More information: <http://manpages.ubuntu.com/manpages/latest/man1/pmount.1.html>.
```
Comment (navarroaxel) [inline pages/linux/pmount.md:4]:
```suggestion
> More information: <https://manpages.ubuntu.com/manpages/latest/man1/pmount.1.html>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages/linux/pmount.md (hunk 13 above)
Comment (vladimyr) [reply pages/linux/pmount.md]: FWIW, `pmount` is maintained by Debian folks so it seems appropriate to use their manpage frontend here.

**Review by schneiderl — APPROVED** [review]

**Comment by marchersimon** [comment]
The commit message should be "use `manned.org` pages where possible". Seems like text in backticks gets cut out.

**Comment by bl-ue** [comment]
If you ran this:
```
git commit -m "use `manned.org` pages where possible"
```
then :smile: it's because your shell tried to execute `manned.org` as a command and replace that with the output from the command.  Did you see something like `command not found: manned.org`?  If so, that confirms it.

Three things you can do:
- Escape the backticks:
  ```
  git commit -m "use \`manned.org\` pages where possible"
  ```

- Use single quotes:
  ```
  git commit -m 'use `manned.org` pages where possible'
  ```

- Write your commit message in the editor that Git opens:
  ```
  git commit
  ```

**Comment by marchersimon** [comment]
Good to know, thanks 😄

**Review by sbrl — APPROVED** [review]
Thanks, @marchersimon!

**Comment by marchersimon** [comment]
Ping @mebeim

**Review by mebeim — APPROVED** [review]
Thanks for the ping @bl-ue. Great to see that comment about `manned.org` on #5143, nice find @vladimyr! Changes look fine to me.

**Review by waldyrious — CHANGES_REQUESTED** [review]
@marchersimon the changes look good, but I'd suggest either (1) removing the changes that are not strictly in the URL (in `pages.fr/linux/ifdown.md` and `pages.zh/linux/logsave.md`), leaving them to a separate PR, or (2) doing an interactive rebase to squash the URL changes into a single commit, and putting those extra changes in a separate PR. Let me know what approach you prefer!

**Comment by marchersimon** [comment]
I prefer the first approach, since that sounds a little easier.

**Comment by waldyrious** [comment]
Hey @marchersimon, sorry if this has been explained elsewhere, but could we make it more explicit why this PR doesn't use manned.org for [`ifup`](https://manned.org/ifup) and [`lftp`](https://manned.org/lftp)? The former is particularly puzzling because manned.org is used for [`ifdown`](https://manned.org/ifdown).

**Comment by marchersimon** [comment]
I didn't use `manned.org` for `ifup`, because <https://manned.org/ifup> has no examples, whereas <https://manpages.debian.org/latest/ifupdown/ifup> does. For `lftp` I didn't use `manned.org`, because it has its own website, and as far as I can remember we agreed on preferring official sources over man pages.

**Comment by bl-ue** [comment]
Oh yeah, the debian `ifup` is much fuller, and from 2017, vs. the manned.org one sourcing a man page from 2009 😝

**Comment by waldyrious** [comment]
Thanks for the context! There's already a bunch of approvals, so I'll go ahead and merge the PR. Sorry that it took so long!

> https://manned.org/ifup has no examples, whereas https://manpages.debian.org/latest/ifupdown/ifup does.

> the debian ifup is much fuller, and from 2017, vs. the manned.org one sourcing a man page from 2009

Hmm, is this something we could contribute or report to the manned.org project? 🤔

**Comment by bl-ue** [comment]
It looks like it's getting it from a CentOS 8.2 distro — https://manned.org/pkg/centos-8.2/BaseOS/network-scripts/10.00.6-1.el8_2.2

I'm not sure that it matters that much.  I think that using the Debian page is fine. 👍🏻

**Comment by waldyrious** [comment]
> I'm not sure that it matters that much. I think that using the Debian page is fine. 👍🏻

What? 😢 it's not about what's in it for us, I meant we could help them improve their own project, which is already pretty impressive :)

(To be clear, I wasn't suggesting holding this PR until they do that, or anything of the sort!)

**Review by waldyrious — APPROVED** [review]
LGTM, thanks @marchersimon!

**Comment by bl-ue** [comment]
>What? 😢 it's not about what's in it for us, I meant we could help them improve their own project, which is already pretty impressive :)

You're right, but to tell the truth, I'm not sure how easy/useful it would be to attempt that, because it's all generated stuff.

**Comment by bl-ue** [comment]
@waldyrious: so you prefer 'multiple pages' to '*'? I personally kinda like the way the asterisk looks ;)

**Comment by waldyrious** [comment]
I'll quote our [guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines) 😉:

> 2. When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness.

**Comment by waldyrious** [comment]
> You're right, but to tell the truth, I'm not sure how easy/useful it would be to attempt that, because it's all generated stuff.

Hence my question — since we're apparently going to rely on them quite heavily going forward, as a default for man page links (but also because it's a pretty cool project all around), I feel it's worth figuring out if we can ensure that potential issues we come across are bubbled up to them — it's just FOSS etiquette, really.

**Comment by waldyrious** [comment]
I've reported this as [issue #1](https://code.blicky.net/yorhel/manned/issues/1) in the manned.org repository! 🎉

**Pull request merged**